src/targets/maven.ts

```diff
@@ -232,7 +235,7 @@ export class MavenTarget extends BaseTarget {
   /**
    * Deletes the user's `gradle.properties` file.
    */
-  public deleteUserGradlePropsFile(): Promise<void> {
+  public async deleteUserGradlePropsFile(): Promise<void> {
```

Comment:
This `async` isn't required, is it?

Reply:
Its not only due to linter setting, but I prefer having explicit `async` when returning a promise. Easier to read imo.